### PR TITLE
Add simple routing closes #37

### LINKED ISSUE #37: Setup react routing
**User Story**
As a frontend developer, I'd like to set up client routing, so that I can navigate between pages.

**Acceptance Criteria**

- React router dom installed.
- React router navigation setup to switch between landing and feed page.
- Skeleton `HomePage.jsx` created with (material ui) button to link to feed page
- Home button in AppBar in feedpage links back to home page
- Unit tests added as appropriate (test routing)

**Notes**

### PR DESCRIPTION
**Pull request by Matteas-Eden**
GitHub Issue (If applicable): #37 

## PR Type

What kind of change does this PR introduce?

- Feature
## What is the current behavior?

No routing present in app.

## What is the new behavior?

Simple routing present in app
- Can go from landing page to feeder page ('login' button)
- Can return to landing from feeder page ('home' icon button)

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current upstream repo
- [x] Added new tests to the test suite for the feature
- [x] Tested the full test suite, and ensured that the feature does not break current build.
- [ ] Docs have been added/updated which fit (for bug fixes / features)
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal)
- [x] Checked licensing of Frameworks/Libraries (if applicable) 

## Other information

Closes #37 

Internal Issue (If applicable):

### CODE REVIEW

File: client/src/pages/FeedPage.jsx [hunk 1]
```diff
@@ -7,7 +7,10 @@ import {
   Paper,
   Grid,
   makeStyles,
+  IconButton,
 } from '@material-ui/core';
+import HomeIcon from '@material-ui/icons/Home';
```
Comment (jordansimsmith) [inline client/src/pages/FeedPage.jsx]: Is it possible to use named exports and object destructuring here?

```javascript
import { HomeIcon } from '@material-ui/icons';
```

Will make more concise import statements if we want multiple icons

File: client/src/pages/FeedPage.jsx [hunk 2]
```diff
@@ -42,7 +49,15 @@ export const FeedPage = () => {
   return (
     <div className={classes.root}>
       <AppBar position="static">
-        <Toolbar>
+        <Toolbar variant="dense">
```
Comment (jordansimsmith) [inline client/src/pages/FeedPage.jsx]: The docs say that 'dense' is desktop only. Can you check how this renders on a mobile browser? 

https://material-ui.com/components/app-bar/

Maybe this should be changed to another variant.

File: client/src/pages/FeedPage.jsx [hunk 3]
```diff
@@ -7,7 +7,10 @@ import {
   Paper,
   Grid,
   makeStyles,
+  IconButton,
 } from '@material-ui/core';
+import HomeIcon from '@material-ui/icons/Home';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
```
Comment (jordansimsmith) [inline client/src/pages/FeedPage.jsx]: `BrowserRouter as Router` can be removed

File: client/src/pages/FeedPage.jsx (hunk 1 above)
Comment (Matteas-Eden) [reply client/src/pages/FeedPage.jsx]: `HomeIcon` isn't exported from `@material-ui/icons`, so I think our hands are tied.

**Pull request merged**